plasmapy/classes/species.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, plasma, particle_type='p', n_particles=1, scaling=1,
                  dt=np.inf * u.s, nt=np.inf):
 
         if np.isinf(dt) and np.isinf(nt):
-            raise ValueError("Both dt and nt are infinite.")
+            raise ValueError("Both dt and nt are infinite.") # coveralls: ignore
```

Comment:
A minor, 100% optional stylistic point...   I usually find the code to look a bit cleaner if the `  # coveralls: ignore` is after the conditional statement.  The coverage reports then ignore the entire clause.
```Python
        if np.isinf(dt) and np.isinf(nt):  # coveralls: ignore
-            raise ValueError("Both dt and nt are infinite.")
```

Reply:
Yeah, I literally missed and hit the wrong line here :D